app/src/main/res/layout/activity_view_record.xml

```diff
@@ -53,7 +53,31 @@
         tools:text="Date" />
 
     <TextView
-        android:id="@+id/textView3"
+        android:id="@+id/record_body_location_header"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="16dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="16dp"
+        android:fontFamily="@font/roboto_bold"
+        android:text="@string/record_body_location"
+        android:textSize="20sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/record_date" />
+
+    <ImageView
+        android:id="@+id/record_body_location"
+        android:layout_width="wrap_content"
```

Comment:
this should be set to a fixed size so that the image can be scaled to that size. (512dp x 512dp)